ai/integrations/langchain/unitycatalog langchain with MLflow example.ipynb

```diff
@@ -0,0 +1,1442 @@
+      "  Created wheel for mlflow: filename=mlflow-2.17.3.dev0-py3-none-any.whl size=5716701 sha256=7cb1b82c144a18cd6c3ce3e6870cdc8b1e82982b000822f977e62f58ef4463ae\n",
+      "  Stored in directory: /tmp/pip-ephem-wheel-cache-gsgjp_yg/wheels/91/bd/b5/77a6d4396adf4c87367f3ed473a71a99791001fb1e4d92adf6\n",
+      "Successfully built mlflow\n",
+      "Installing collected packages: zipp, wrapt, Werkzeug, sqlparse, smmap, markdown, Mako, itsdangerous, gunicorn, graphql-core, cloudpickle, click, blinker, importlib-metadata, graphql-relay, gitdb, Flask, docker, deprecated, alembic, opentelemetry-api, graphene, gitpython, opentelemetry-semantic-conventions, opentelemetry-sdk, mlflow\n",
+      "  Attempting uninstall: zipp\n",
+      "    Found existing installation: zipp 1.0.0\n",
+      "    Not uninstalling zipp at /usr/lib/python3/dist-packages, outside environment /local_disk0/.ephemeral_nfs/envs/pythonEnv-dec52b04-7b0e-4c7a-a792-37744b0041b8\n",
+      "    Can't uninstall 'zipp'. No files were found to uninstall.\n",
+      "  Attempting uninstall: click\n",
+      "    Found existing installation: click 8.0.4\n",
+      "    Not uninstalling click at /databricks/python3/lib/python3.10/site-packages, outside environment /local_disk0/.ephemeral_nfs/envs/pythonEnv-dec52b04-7b0e-4c7a-a792-37744b0041b8\n",
+      "    Can't uninstall 'click'. No files were found to uninstall.\n",
+      "  Attempting uninstall: blinker\n",
+      "    Found existing installation: blinker 1.4\n",
+      "    Not uninstalling blinker at /usr/lib/python3/dist-packages, outside environment /local_disk0/.ephemeral_nfs/envs/pythonEnv-dec52b04-7b0e-4c7a-a792-37744b0041b8\n",
+      "    Can't uninstall 'blinker'. No files were found to uninstall.\n",
+      "  Attempting uninstall: importlib-metadata\n",
+      "    Found existing installation: importlib-metadata 4.6.4\n",
+      "    Not uninstalling importlib-metadata at /usr/lib/python3/dist-packages, outside environment /local_disk0/.ephemeral_nfs/envs/pythonEnv-dec52b04-7b0e-4c7a-a792-37744b0041b8\n",
+      "    Can't uninstall 'importlib-metadata'. No files were found to uninstall.\n",
+      "Successfully installed Flask-3.0.3 Mako-1.3.6 Werkzeug-3.1.2 alembic-1.14.0 blinker-1.8.2 click-8.1.7 cloudpickle-3.1.0 deprecated-1.2.14 docker-7.1.0 gitdb-4.0.11 gitpython-3.1.43 graphene-3.4.1 graphql-core-3.2.5 graphql-relay-3.2.0 gunicorn-23.0.0 importlib-metadata-8.5.0 itsdangerous-2.2.0 markdown-3.7 mlflow-2.17.3.dev0 opentelemetry-api-1.28.0 opentelemetry-sdk-1.28.0 opentelemetry-semantic-conventions-0.49b0 smmap-5.0.1 sqlparse-0.5.1 wrapt-1.16.0 zipp-3.20.2\n",
+      "\u001b[43mNote: you may need to restart the kernel using %restart_python or dbutils.library.restartPython() to use updated packages.\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install unitycatalog-ai[databricks] unitycatalog-langchai langchain_openai langgraph mlflow\n",
```

Comment:
```suggestion
    "%pip install -qqq unitycatalog-ai[databricks] unitycatalog-langchain langchain_openai langgraph mlflow\n",
```
Just so we don't have to see the wall of pip installation text in the notebook :) (also minor typo)